### ISSUE TITLE
ApiKey is URL encoded in Swagger UI which breaks "Bearer" authentication.

### ISSUE THREAD

**Issue by urig** [domaindrivendev/Swashbuckle#851]
This issue is based on this question on Stackoverflow.com: [Enable bearer token in Swashbuckle (Swagger document)](http://stackoverflow.com/questions/39009758/enable-bearer-token-in-swashbuckle-swagger-document/39083615)

### Steps to reproduce:
1. In an ASP.net Web API project where Swashbuckle is installed, setup ApiKey authentication like so:  

    ```c#
    GlobalConfiguration.Configuration  
        .EnableSwagger(c =>  
        {  
            c.ApiKey("Token")
                .Description("Filling bearer token here")
                .Name("Authorization")
                .In("header");
        }
        .EnableSwaggerUi(c =>
        {
            c.EnableApiKeySupport("Authorization", "header");
        };
    ```
2. Run the project and browse over to the Swagger UI ("/swagger/ui/index")  

3. Enter a valid "Bearer" header value into the "ApiKey" input field (top left corner). For example:
> Bearer 78898af0-5fb4-44b2-833d-fc8be51750c3z  

4. Invoke a controller action

### Actual behavior:
The request goes out with an HTTP header like so:
> Authorization -> Bearer%2078898af0-5fb4-44b2-833d-fc8be51750c3  

It is therefore not parsed correctly into Web API's `HttpAuthenticationContext.Request.Headers.Authorization`.

### Expected behavior:
The request should go out with an HTTP header like so:
> Authorization -> Bearer 78898af0-5fb4-44b2-833d-fc8be51750c3  

In which case `HttpAuthenticationContext.Request.Headers.Authorization` has `Scheme` correctly set to "Bearer" and `Parameter` correctly set to "78898af0-5fb4-44b2-833d-fc8be51750c3z".

### Root cause:
I believe the root cause is this line in [CustomAssets/index.html](https://github.com/domaindrivendev/Swashbuckle/blob/master/Swashbuckle.Core/SwaggerUi/CustomAssets/index.html#L117):

    var key = encodeURIComponent($('#input_apiKey')[0].value);

**Comment by vinod90angular** [comment]
In swagger i have given same as above mentioned GlobalConfiguration.Configuration  
    .EnableSwagger(c =>  
    {  
        c.ApiKey("Token")
            .Description("Filling bearer token here")
            .Name("Authorization")
            .In("header");
    }
    .EnableSwaggerUi(c =>
    {
        c.EnableApiKeySupport("Authorization", "header");
    };
but it's not working properly  And i pass Header key value is "Token".Can you please give solution

**Comment by OzBob** [comment]
@vinod90angular It was fixed in this pull request made on Aug 23 2016: https://github.com/domaindrivendev/Swashbuckle/commit/0792d4b8cba61b26df4fe4edf0d930bdbcdd3d9a

Which is tagged '5.60' but I can't tell whether it's been included in 5.6.0 or not. 

You could get the source and build this project yourself to ensure that bit of javascript is present ...

Maybe @domaindrivendev can tell us … coz I also couldn't see it in the new Core project's:  https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/a233b94a00a8ab629a20e68ecdce22b56bf62928/src/Swashbuckle.AspNetCore.SwaggerUI/index.html
